python/test.sh

```diff
@@ -42,7 +42,7 @@ python3 $FF_HOME/examples/python/onnx/mnist_mlp_pt.py
 ./flexflow_python $FF_HOME/examples/python/keras/seq_reuters_mlp.py -ll:py 1 -ll:gpu $GPUS -ll:fsize $LEGION_FSIZE_SMALL -ll:zsize $LEGION_ZSIZE
```

Comment:
Is this the file we should be running? There seems to be some question in #362. Sounds like we should be running multi-gpu tests instead?